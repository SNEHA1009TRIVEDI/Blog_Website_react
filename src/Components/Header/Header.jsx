//create a react compoent
import React from "react";
import "./Header.css";

const Header = () => {
  return (
    
      <nav className='body-padding'>
      <div>
        <h1>The Artifact</h1>
        <p>
          <em>Culture & Art blog</em>
        </p>
      </div>
      <ul>
        <li>
          <a href='/blog'>Blog</a>
        </li>
        <li>
          <a href='/about'>About</a>
        </li>
        <li>
          <a href='/contact'>Contact</a>
        </li>
      </ul>
    </nav>
    
  );
};

export default Header;
