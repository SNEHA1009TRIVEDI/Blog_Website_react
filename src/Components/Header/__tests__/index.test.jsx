import { render } from "@testing-library/react";
import React from "react";

import Header from "../Header.jsx";
describe("Header", () => {
  it("should renders without crashing ", () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
