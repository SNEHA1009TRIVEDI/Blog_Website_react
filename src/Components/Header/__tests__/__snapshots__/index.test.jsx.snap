// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should renders without crashing  1`] = `
<DocumentFragment>
  <nav
    class="body-padding"
  >
    <div>
      <h1>
        The Artifact
      </h1>
      <p>
        <em>
          Culture & Art blog
        </em>
      </p>
    </div>
    <ul>
      <li>
        <a
          href="/blog"
        >
          Blog
        </a>
      </li>
      <li>
        <a
          href="/about"
        >
          About
        </a>
      </li>
      <li>
        <a
          href="/contact"
        >
          Contact
        </a>
      </li>
    </ul>
  </nav>
</DocumentFragment>
`;
